### PR TITLE
Im package readme update

### PR DESCRIPTION
**Pull request by princerb**

### CODE REVIEW

File: package.json
Comment (TangoIndiaMango) [inline package.json:1]: Why do we have empty version?! Is that ideal?

Comment (princerb) [reply package.json:1]: package.json file is for `cdnjs`, so we will change it when we will be first ready to upload to `cdnjs`,

Comment (princerb) [reply package.json:1]: like for the production version.

Comment (TangoIndiaMango) [reply package.json:1]: Okay. Thanks for clarifying that

**Review by TangoIndiaMango — APPROVED** [review]

File: package.json
Comment (princerb) [reply package.json:1]: 👍

**Pull request merged**